### ISSUE TITLE
Python 2.7 compatibility

### ISSUE THREAD

**Issue by FlorianWilk** [FlorianWilk/SpotMicroAI#3]
Interesting... when running
python3 gamepad.py  
everything works. The Bot falls to the ground and keeps standing.
When i use python 2.7:
python gamepad.py
The Bot falls down. The left and right legs have different positions. Why is that?!

**Comment by littleggghost** [comment]
I use python2.7 now, here is the initial pose:
![Screenshot from 2019-05-27 19-47-33](https://user-images.githubusercontent.com/5058791/58417801-85e18d00-80b8-11e9-9eca-94d1d5dbcdd8.png)

**Comment by FlorianWilk** [comment]
Yes, same here. I will keep on using Python3 to have some progress on the
Kinematics.
Maybe you want to investigate this bug? I have no clue why Python2.7 should
do different calculations than Python3 does.

**Comment by littleggghost** [comment]
Maybe the initial command caused this. 
You can try this initial pose in gamepad.py:
`line224    angles = kin.calcIK(Lp, (math.pi/180*roll, 0, 0), (100/256*-joy_rz-20+100, 40+height, joy_x))`
